### DISCUSSION TITLE
Training the boundary

### DISCUSSION THREAD

**Discussion by SuchitReddi** [yzyouzhang/AIR-ASVspoof#39] [General]
In the paper, it's mentioned that:

`The key idea of one-class classification methods is to capture the target class distribution and set a tight classification boundary around it, so that all non-target data would be placed outside the boundary.`

What I understood was that we take bona fide speech and to train and create a tight boundary around it using oc-softmax loss function.

But when I saw the ASVspoof dataset, in LA cm protocols, the train file has bona fide and also spoof audio files.

So, what I can't conclude is, does one class classification mean we only take the bona fide speech for making a classified boundary?

I am confused about how we train the boundary. Using only bona fide speech data or using both bona fide and spoof data.

Can you clarify this for me @yzyouzhang ?

**Comment by yzyouzhang** [comment]
We use both classes to train. The idea of one-class learning is to make the bonafide representation compact and separate the spoofing attacks far away from the boundary. This is consistent with [1] in their third case of definition, where the negative class is not statistically representative in the training data.

[1] Khan, S., & Madden, M. (2014). One-class classification: Taxonomy of study and review of techniques. The Knowledge Engineering Review, 29(3), 345-374. doi:10.1017/S026988891300043X

**Reply by SuchitReddi** [reply]
So, that means the power of OCC is more evident if the spoofed data is nearly non-existing. Did I understand it correctly?

**Reply by yzyouzhang** [reply]
I don't think so. It's hard to say when OCC is more evident. It's only about solving problems with OCC. The anti-spoofing problem is a good fit because of the generalization issue.

**Reply by SuchitReddi** [reply]
Ok, so we take the best features of Softmax and AM-Softmax and create a compact and tight boundary for target data resulting in less target data being falsely identified as fake, and loose boundary for non-target data, resulting in more unknown attacks being caught in the non-target boundary. Right?

**Reply by yzyouzhang** [reply]
Yes.

**Reply by SuchitReddi** [reply]
Thank you very much for replying this quick!

**Reply by SuchitReddi** [reply]
It would be helpful if you can provide the presentation ppt you used for making your youtube video. Thank you.

**Reply by yzyouzhang** [reply]
Here is an updated version of my slides. https://labsites.rochester.edu/air/publications/ICASSP2022Slides_Neil.pdf

**Reply by SuchitReddi** [reply]
Thanks
